services/users/profile_picture.go

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package users
+
+import (
+	"bytes"
+	"hash/fnv"
+	"image"
+	"image/color"
+	"image/draw"
+	"image/png"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/golang/freetype"
+	"github.com/golang/freetype/truetype"
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/utils/fileutils"
+)
+
+const (
+	ImageProfilePixelDimension = 128
+)
+
+func (us *UserService) GetDefaultProfileImage(user *model.User) ([]byte, error) {
+	if user.IsBot {
+		return model.BotDefaultImage, nil
```

Comment:
Let's move `model.BotDefaultImage` as an unexported variable inside this package. It's only used here.

Reply:
It seems to be used in EE code for testing 🤷

Reply:
I think any random test image would work there. I just tested with tests/test.png and it worked. But we can do it later 0/5.